app/Http/Controllers/ClientController.php

```diff
@@ -147,6 +154,8 @@ public function show(ClientRequest $request)
      */
     public function create(ClientRequest $request)
     {
+        Auth::user()->can('create', ENTITY_CLIENT);
```

Comment:
Not a big deal, it can be left as is. But, fyi... a cleaner approach may be to change the request from ClientRequest to CreateClientRequest as it would handle the authorization check.